### PR TITLE
Add standalone sprite builder

### PR DESCRIPTION
**Pull request by janisozaur**
This is towards addressing https://github.com/OpenRCT2/OpenRCT2/issues/15191 given recent influx of Android users.

This only requires libpng (and transitively zlib) on host and nlohmann-json, which is a header only library which we ship in Android docker images already.

Next step is to update cmake to find libpng, verify it is installed in relevant docker, modify build steps for android to compile g2, embed the resulting g2 (and probably `data/`) somewhere in the apk.

### CODE REVIEW

**Comment by janisozaur** [comment]
https://github.com/OpenRCT2/openrct2-docker-build/blob/ccec17a6bb7daec974faf0d49c0607852553c9be/0.3.1/android/Dockerfile#L14

`libpng-dev` is part of the image already

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -349,8 +349,14 @@ jobs:
...
+            cmake .. -G Ninja
+            cmake --build .
+            ./sb build ../../../g2.dat ../../../../../resources/g2/sprites.json
+          popd
           pushd src/openrct2-android
-            ./gradlew app:assemblePR
+            ./gradlew app:assemble
           popd
           mkdir -p artifacts
           mv src/openrct2-android/app/build/outputs/apk/arm/pr/app-arm-pr.apk artifacts/openrct2-arm.apk
```
Comment (janisozaur) [inline .github/workflows/ci.yml:362]:
```suggestion
          find . -name '*.apk'
          mv src/openrct2-android/app/build/outputs/apk/arm/pr/app-arm-pr.apk artifacts/openrct2-arm.apk
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -349,10 +349,17 @@ jobs:
...
+            cmake .. -G Ninja
+            cmake --build .
+            ./sb build ../../../g2.dat ../../../../../resources/g2/sprites.json
+          popd
           pushd src/openrct2-android
-            ./gradlew app:assemblePR
+            ./gradlew app:assemble
           popd
           mkdir -p artifacts
+          find . -name '*.apk'
```
Comment (janisozaur) [inline .github/workflows/ci.yml:362]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -349,10 +349,17 @@ jobs:
...
           mv src/openrct2-android/app/build/outputs/apk/arm/pr/app-arm-pr.apk artifacts/openrct2-arm.apk
```
Comment (janisozaur) [inline .github/workflows/ci.yml:363]:
```suggestion
          mv src/openrct2-android/app/build/outputs/apk/release/app-release.apk artifacts/openrct2-arm.apk
```

**Comment by IntelOrca** [comment]
Now available from here:
https://github.com/IntelOrca/libsawyer/releases

**Pull request closed without merging**